lib/browser/api/menu-item-roles.js

```diff
@@ -154,6 +154,69 @@ const roles = {
         webContents.setZoomLevel(zoomLevel - 0.5)
       })
     }
+  },
+  // submenu Edit (should fit both Mac & Windows)
+  editMenu: {
+    label: 'Edit',
+    submenu: [
+      {
+        role: 'undo'
+      },
+      {
+        role: 'redo'
+      },
+      {
+        type: 'separator'
+      },
+      {
+        role: 'cut'
+      },
+      {
+        role: 'copy'
+      },
+      {
+        role: 'paste'
+      },
+
+      process.platform === 'darwin' ? {
+        role: 'pasteandmatchstyle'
+      } : {},
+
+      {
+        role: 'delete'
+      },
+
+      process.platform === 'win32' ? {
+        type: 'separator'
+      } : {},
+
+      {
+        role: 'selectall'
+      }
+    ]
+  },
+
+  // submenu Window should be used for Mac only
+  windowMenu: {
+    label: 'Window',
+    submenu: [
+      {
+        role: 'minimize'
+      },
+      {
+        role: 'close'
+      },
+
+      process.platform === 'darwin' ? {
+        type: 'separator'
+      } : {},
+
+      process.platform === 'darwin' ? {
+        label: 'Bring All to Front',
```

Comment:
Is this label needed here? Shouldn't it use the one defined by the `front` role if unspecified?